internal/services/postgres/postgresql_flexible_server_resource_test.go

```diff
@@ -604,6 +608,59 @@ func TestAccPostgresqlFlexibleServer_publicNetworkAccessEnabled(t *testing.T) {
 	})
 }
 
+func TestAccPostgresqlFlexibleServer_writeOnlyPassword(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_postgresql_flexible_server", "test")
+	r := PostgresqlFlexibleServerResource{}
+
+	resource.ParallelTest(t, resource.TestCase{
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.SkipBelow(version.Must(version.NewVersion("1.11.0-beta1"))),
```

Comment:
Should this be `1.11.0`?
```suggestion
			tfversion.SkipBelow(version.Must(version.NewVersion("1.11.0"))),
```